### DISCUSSION TITLE
Possibly add the function to be able to custom left click "wield" default wieldable items?

### DISCUSSION THREAD

**Discussion by HardMoo** [runelite/runelite#16281] [Ideas]
The specific example I came across is when I have the "Teleport item" enabled under menu entry swapper, I am unable to wield my wilderness sword 4 without right clicking. I thought "great, I can just customize the left click and make it easily wieldable." Only to find out I couldn't. I like to have the "teleport Item" function enabled for items such as glories and rings of dueling. The wilderness sword is one item that I find myself equipping more than actually using the teleport. I would like to be able to do so, without having to customize left click for all of my "teleport items". The Wilderness sword 4 is a specific example of where current functionality between "teleport item" and "customizable left click" feels... clunky.

**Comment by Alexsuperfly** [comment]
because the swap is done after shift clicking (i think), it still believes wield is the top option and so it does not offer that to select.  You can however select another option and then select wield and it will allow you to have wield on top.

**Reply by HardMoo** [reply]
That's the issue I'm having. even after swapping my left click to say "drop", "teleport", or "examine".. I still dont have "Wield" as a customizable left click.
